hw3/01-characters.js

```diff
@@ -1,4 +1,59 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
+section.classList.add('d-flex', 'flex-row', 'flex-wrap', 'justify-content-around', 'mt-4');
 
-fetch(url);
+function addCharacters(characters) {
+  characters.forEach((character) => {
+    const charCard = document.createElement('div');
+    const image = document.createElement('img');
+    const textBox = document.createElement('div');
+    const fullName = document.createElement('h4');
```

Comment:
You're skipping from an h1 header to an h4 which is causing an accessibility issue. Maybe make this h2 instead?